src/index.html

```diff
@@ -7,6 +7,333 @@
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+              </div>
+              <p class="card__comments">Отзывов: 5</p>
+            </div>
+            <div class="card__price">
+              <p class="card__price-hint">Цена:</p>
+              <p class="card__price-value">69 999 грн</p>
+            </div>
+            <a
+              href="/"
+              class="card__buy-link"
+            >
+              <div class="card__buy">
+                Купить
+              </div>
+            </a>
+          </div>
+          <div class="card catalog__card">
+            <img
+              src="images/imac.jpeg"
+              alt="APPLE A1419 iMac 27 Retina 5K"
+              class="card__img"
+            >
+            <h1 class="card__title">Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)</h1>
+            <p class="card__product-code">Код товара: 195434</p>
+            <div class="card__reiting">
+              <div class="stars stars--4">
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+              </div>
+              <p class="card__comments">Отзывов: 5</p>
+            </div>
+            <div class="card__price">
+              <p class="card__price-hint">Цена:</p>
+              <p class="card__price-value">69 999 грн</p>
+            </div>
+            <a
+              href="/"
+              class="card__buy-link"
+            >
+              <div class="card__buy">
+                Купить
+              </div>
+            </a>
+          </div>
+          <div class="card catalog__card">
+            <img
+              src="images/imac.jpeg"
+              alt="APPLE A1419 iMac 27 Retina 5K"
+              class="card__img"
+            >
+            <h1 class="card__title">Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)</h1>
+            <p class="card__product-code">Код товара: 195434</p>
+            <div class="card__reiting">
+              <div class="stars stars--4">
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+              </div>
+              <p class="card__comments">Отзывов: 5</p>
+            </div>
+            <div class="card__price">
+              <p class="card__price-hint">Цена:</p>
+              <p class="card__price-value">69 999 грн</p>
+            </div>
+            <a
+              href="/"
+              class="card__buy-link"
+            >
+              <div class="card__buy">
+                Купить
+              </div>
+            </a>
```

Comment:
```suggestion
            <a href="/" class="card__buy-link">                           
               Купить
            </a>
```
you don't have to use div (in my view)